test/hack/resource/clean/main.go

```diff
@@ -26,13 +26,18 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/iam"
 	"github.com/samber/lo"
 	"go.uber.org/zap"
+	"golang.org/x/exp/slices"
 
 	"github.com/aws/karpenter-provider-aws/test/hack/resource/pkg/metrics"
 	"github.com/aws/karpenter-provider-aws/test/hack/resource/pkg/resourcetypes"
 )
 
 const sweeperCleanedResourcesTableName = "sweeperCleanedResources"
 
+var excludedClusters = []string{
+	"soak-periodic-46287782",
```

Comment:
TODO statement to remove this from the file once we are done with the investigation